src/main/scala/outwatch/dom/ValueObservable.scala

```diff
@@ -3,22 +3,29 @@ package outwatch.dom
 import cats.Functor
 import monix.reactive.Observable
 
-trait ValueObservable[+T] {
+trait ValueObservable[+T] { self =>
```

Comment:
This trait is only talking about the initial value, isn't it? Took me some moments to figure that out. Should we rename it, or write a comment?

Reply:
Good point. A comment would definitely help. And I am investigating whether we can have `type ProHandler = ValueObservable with Observer` instead of `Observable with Observer`. This allows for less update coming from handler. We can implement a Handler similar to the `BehaviorSubject` which tracks the last emitted value of the stream and then provides some common operators to transform a ValueObservable similar to an Observable.

Any ideas for a better name? It is an Observable with an initial/current value.

Reply:
`ObservableWithInitialValue` :wink:  ?

Reply:
Let's leave this part for #254 where we are anyhow touching the ValueObservable type. This type is already on master, so we can check there for renaming too.